include/OpenSpaceToolkit/Mathematics/Geometry/3D/Transformation/Rotation/Quaternion.hpp

```diff
@@ -275,7 +275,7 @@ class Quaternion
     ///
     /// @return                 True if Quaternion is unitary
 
-    bool isUnitary() const;
+    bool isUnitary(const Real& aNormTolerance = Real::Epsilon()) const;
```

Comment:
This may introduce an ABI break for consumers of the library, we should instead overload `isUnitary() ` by declaring a new `bool isUnitary(const Real& aNormTolerance)`

Reply:
This is a good point, but, unfortunately, ABI breaks aren't really something we track at the moment, so there is a good chance we are already breaking the ABI with many of our releases :(

Reply:
Really good point @youssefahammou. Indeed I think it's fine for now but something we should think about in the future